app/main.py

```diff
@@ -2,0 +1,21 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        print("Invalid command format. Use: cp <source> <destination>")
```

Comment:
The error message is clear and informative, which is good practice. Ensure consistency in using double quotes for strings, as recommended in the checklist.